demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -114,7 +118,14 @@ def generate_integration_doc(
         else:
             docs = []  # type: list
             docs.extend(add_lines(yml_data.get('description')))
-            docs.extend(['This integration was integrated and tested with version xx of {}'.format(yml_data['name'])])
+            docs.extend(['This integration was integrated and tested with version xx of {}'.format(yml_data['name']), ''])
+            # Checks if the integration is a new version
+            integration_version = re.findall("[vV][2-9].yml$", input_path)
```

Comment:
why not use the is v2 method from tools? if it does not detect things above v2 lets update the method and still use it here.

Reply:
done.